pkg/cmd/controller/standalone_apiserver.go

```diff
@@ -47,7 +48,7 @@ func RunControllerServer(servingInfo configv1.HTTPServingInfo, kubeExternal clie
 		return err
 	}
 	sarClient := kubeExternal.AuthorizationV1()
-	remoteAuthz, err := authzwebhook.NewFromInterface(sarClient, 5*time.Minute, 5*time.Minute, *authzwebhook.DefaultRetryBackoff(), authzwebhook.AuthorizerMetrics{
+	remoteAuthz, err := authzwebhook.NewFromInterface(sarClient, 5*time.Minute, 5*time.Minute, *authzwebhook.DefaultRetryBackoff(), authorizer.DecisionNoOpinion, authzwebhook.AuthorizerMetrics{
```

Comment:
using the original default behaviour https://github.com/kubernetes/kubernetes/commit/2e2f51a4417d93b5505091d28b319365dc95e137#diff-8b41b5e2c7d3bb27fa452e58ab0a4120b02d3a7a16d8961954f5c3c270ab8935L125